dspace-api/src/main/java/org/dspace/app/canvasdimension/CanvasDimensionCLI.java

```diff
@@ -0,0 +1,202 @@
+
+
+        boolean iiifEnabled = configurationService.getBooleanProperty("iiif.enabled");
+        if (!iiifEnabled) {
+            System.out.println("WARNING: IIIF is not enabled on this DSpace server.");
+        }
+
+        // default to not updating existing dimensions
+        boolean force = false;
+        // default to printing messages
+        boolean isQuiet = false;
+        // default to no limit
+        int max2Process = Integer.MAX_VALUE;
+
+        String identifier = null;
+        String eperson = null;
+
+        Context context = new Context();
+        IIIFCanvasDimensionService canvasProcessor = IIIFCanvasDimensionServiceFactory.getInstance()
+                                                                                      .getIiifCanvasDimensionService();
+
+        CommandLineParser parser = new DefaultParser();
+
+        Options options = new Options();
+        options.addOption("i", "identifier", true,
+            "process IIIF canvas dimensions for images belonging to this identifier");
+        options.addOption("e", "eperson", true,
+            "email of eperson setting the canvas dimensions");
+        options.addOption("f", "force", false,
+            "force update of all IIIF canvas height and width dimensions");
+        options.addOption("q", "quiet", false,
+            "do not print anything except in the event of errors.");
+        options.addOption("m", "maximum", true,
+            "process no more than maximum items");
+
+        Option skipOption = Option.builder("s")
+                                  .longOpt("skip")
+                                  .hasArg()
+                                  .hasArgs()
+                                  .valueSeparator(',')
+                                  .desc(
+                                      "SKIP the bitstreams belonging to identifier\n" +
+                                          "Separate multiple identifiers with a comma (,)\n" +
+                                          "(e.g. -s \n 123456789/34,123456789/323)")
+                                  .build();
+        options.addOption(skipOption);
+
+        CommandLine line = null;
+
+        try {
+            line = parser.parse(options, argv);
+        } catch (MissingArgumentException e) {
+            System.out.println("ERROR: " + e.getMessage());
+            HelpFormatter help = new HelpFormatter();
+            help.printHelp("CanvasDimension processor\n", options);
+            System.exit(1);
+        }
+
+        if (line.hasOption('f')) {
+            force = true;
+        }
+        if (line.hasOption('q')) {
+            isQuiet = true;
+        }
+        if (line.hasOption('e')) {
+            eperson = line.getOptionValue('e');
+        }
+        if (line.hasOption('i')) {
+            identifier = line.getOptionValue('i');
+        } else {
+            System.out.println("An identifier for a Community, Collection, or Item must be provided.");
```

Comment:
Can we print out the help text here as well?  The first time I ran this with no arguments, I expected it to error out, but also tell me what a valid argument would look like.  

There's also no "--help" or "-h" option...so, it's impossible to see the list of valid arguments from commandline.  We should add a "-h" option